core/types/block.go

```diff
@@ -114,6 +114,9 @@ type Header struct {
 	extra                    []byte        `json:"extraData"             gencodec:"required"`
 	stateLimit               uint64        `json:"stateLimit" 			  gencodec:"required"`
 	stateUsed                uint64        `json:"stateUsed"             gencodec:"required"`
+	exchangeRate             *big.Int      `json:"exchangeRate"		  gencodec:"required"`
+	quaiToQi                 *big.Int      `json:"quaiToQi"			  gencodec:"required"`
```

Comment:
Why do you put both of these in the header? won't they always be the inverse of each other?